packages/apps/docs/package.json

```diff
@@ -15,7 +15,7 @@
     "build:importReadMe": "node ./src/scripts/importReadme.mjs && prettier ./src/pages/docs --write",
     "build:next": "pnpm run build:scripts && next build",
     "build:scripts": "pnpm run build:importReadMe && pnpm run build:createtree && pnpm run build:createSpecJson && pnpm run build:brokenLinks && pnpm run build:checkForHeaders",
-    "build:storybook": "storybook build",
+    "build:storybook": "pnpm run build:createtree && storybook build",
```

Comment:
this will already run when the normal build will run.
Normal build is necessary in the monorepo to kick off dependency builds like `react-ui`

Reply:
with turbo repo we can build all dependencies separately without running complete build. So, that's what we're doing as part of chromatic ci. So, we need this script separately.